src/trading/universal_trader.py

```
@@ -457,31 +382,16 @@ async def _handle_token(self, token_info: TokenInfo) -> None:
 
             # Only wait for next token in yolo mode
             if self.yolo_mode:
-                logger.info(
-                    f"YOLO mode enabled. Waiting {self.wait_time_before_new_token} seconds before looking for next token..."
-                )
+                logger.info(f"YOLO mode enabled. Waiting {self.wait_time_before_new_token} seconds before looking for next token...")
                 await asyncio.sleep(self.wait_time_before_new_token)
 
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)